Taudem5PCVS2010/FlowdirConditioning/FlowdirConditioning.vcxproj

```diff
@@ -0,0 +1,150 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@dtarb Is this project needed? It seems this branch has 2 new projects - FlowdirCond and FlowdirConditioning